pretty_midi/pretty_midi.py

```diff
@@ -71,6 +81,24 @@ def __init__(self, midi_file=None, resolution=220, initial_tempo=120.):
                                "  This is not a valid type 0 or type 1 MIDI "
                                "file.  Timing may be wrong."), RuntimeWarning)
 
+            # Check that there are only key signature change events on track 0
+            if sum([sum([isinstance(event, midi.events.KeySignatureEvent)
```

Comment:
Let's combine these three checks into a single one - `isinstance` can take a `tuple` as its second argument, so the call would look like `isinstance(event, (midi.events.SetTempoEvent, midi.events.KeySignatureEvent, midi.events.TimeSignatureEvent)` and you could change the warning to say something like "Key signature, set tempo, or time signature events found on non-zero...".